sql/src/main/java/io/crate/metadata/table/ColumnRegistrar.java

```diff
@@ -143,18 +162,22 @@ private void registerPossibleObjectInnerTypes(String topLevelName, List<String>
                 null
             );
             pos++;
-            infosBuilder.put(ref.column(), ref);
+            if (!infos.containsKey(ref.column())) {
```

Comment:
does it make sense to use `{put/compute}IfAbsent` methods here?

Reply:
Yes, indeed. Thanks!